src/index.js

```diff
@@ -126,6 +128,10 @@ export default function Loadable<Props: {}, Err: Error>(opts: Options) {
     render() {
       let { pastDelay, error, Component } = this.state;
 
+      if (!isWebpack && serverSideRequirePath) {
```

Comment:
I agree on this count. I was able to get this working with Webpack by removing the `!isWebpack` check and following a slightly modified version of the pattern @thejameskyle adumbrated [here](https://gist.github.com/thejameskyle/abecfe8ec2a7ce1e312a904527a31908) server-side (which you can see [here](https://gist.github.com/phyllisstein/a33964c7f393d7f846c3b11fe1659a76), if you're curious). After making those two changes, I have SSR working reliably.